content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -10,23 +10,45 @@ feature:
 
 {{% capture overview %}}
 
-When you specify a [Pod](/docs/concepts/workloads/pods/pod/), you can optionally specify how
-much CPU and memory (RAM) each Container needs. When Containers have resource
-requests specified, the scheduler can make better decisions about which nodes to
-place Pods on. And when Containers have their limits specified, contention for
-resources on a node can be handled in a specified manner. For more details about
-the difference between requests and limits, see
-[Resource QoS](https://git.k8s.io/community/contributors/design-proposals/node/resource-qos.md).
+When you specify a {{< glossary_tooltip term_id="pod" >}}, you can optionally specify how
+much resources each {{< glossary_tooltip text="Container" term_id="container" >}} needs.
+The most common resources you specify are CPU and memory (RAM), and there are others you might
+use as well.
+
+When you specify the resource _request_ for Containers, the scheduler can make better decisions
+about which nodes to place Pods on. And when you specify _limits_ for Containers, that lets
+the kubelet on a node manage contention for those resources.
+
+
 
 {{% /capture %}}
 
 
 {{% capture body %}}
 
+## Requests and limits
+
+A Container can exceed its resource request if the Node has enough of that resource available.
+But a Container is not allowed to use more than its resource limit limit.
+
+For example, if you set a request of 256 MiB for a container, and that Container is in a Pod
```

Comment:
Revision, but: 
> For example, if you request 256 MiB for a container, and that Container is in a Pod


... 256 MiB of what? RAM?

Reply:
Ah, this might work better:
```markdown
For example, if you set a `memory` request of 256 MiB for a container, and that Container is in a Pod
```